### PR TITLE
Enable encryption in json files and accept additional scopes in URL Style

### PR DESCRIPTION
**Pull request by nasatome**
For those situations where some of your collaborators have access to the server, but they do not have access to the environment variable of laravel, since they are dockerized and do not have access to those images, but they do have access to Storage.

### CODE REVIEW

**Comment by nasatome** [comment]
i added new commits in the master branch for testing (in my dev-master composer file), I did not know that github took all the changes even if you did not do PullRequest,

in short, I thought to do a second PullRequest about accepting additional Scopes
https://github.com/dacastro4/laravel-gmail/pull/65/commits/17cd021e1105ffd2e13961c763d9ac5c94ba21b4

I hope you accept the commits, I need them, and I hope they serve more people.

excellent work!

**Comment by nasatome** [comment]
By the way, note that the update of docs that I made.
https://github.com/dacastro4/laravel-gmail/pull/65/commits/9647db360d27f77215dd6a26163b1f338a6cc023
 fix #46

Some do not know how to call a service.

File: src/config/gmail.php [hunk 1]
```diff
@@ -41,6 +41,23 @@
...
+	|
+	|   'additional_scopes' => [
+	|        'https://www.googleapis.com/auth/drive',
+	|        'https://www.googleapis.com/auth/documents'
+	|   ],
+	|
+	|
+	*/
+
+    'additional_scopes' => [
```
Comment (dacastro4) [inline src/config/gmail.php]: I don't understand this part here, why did you add `additional_scopes` when you have `scopes` already.

File: src/Traits/Configurable.php [hunk 2]
```diff
@@ -106,7 +113,7 @@ private function mapScopes()
 			}
 		}
 
-		return $mappedScopes;
+		return  array_merge( $mappedScopes , $this->_config[ 'gmail.additional_scopes' ] );
```
Comment (dacastro4) [inline src/Traits/Configurable.php]: Following from the config file `additional_scopes`, I'm already doing this but with the regular scopes

**Comment by nasatome** [comment]
The reason is to allow with the same token access to other API tools, such as
Google DRIVE, Google SpreadSheets, Google Slides, etc

 'additional_scopes' => [
            'https://www.googleapis.com/auth/drive',
            'https://www.googleapis.com/auth/documents',
            'https://www.googleapis.com/auth/spreadsheets'
    ],

I fully understand that they are outside the scope of the project, which is access to the GMAIL API, but this gives the project flexibility in certain cases.
I would very much appreciate that you allow access to other APIs in addition to GMAIL's.

**Comment by nasatome** [comment]
Now I ask, the "setAdditionalScopes" function can include the Google Drive API, Google Documents or Google Spreadsheets in some way? an apology if I did not realize it, but I did not see how this would integrate

**Comment by nasatome** [comment]
@dacastro4  


This refers to obtaining these other permits.

File: src/config/gmail.php (hunk 1 above)
Comment (nasatome) [reply src/config/gmail.php]: https://github.com/dacastro4/laravel-gmail/pull/65#issuecomment-486810224

File: src/Traits/Configurable.php (hunk 2 above)
Comment (nasatome) [reply src/Traits/Configurable.php]: https://github.com/dacastro4/laravel-gmail/pull/65#issuecomment-486811868
